concourse/pipeline/guest-package-build.yaml

```diff
@@ -0,0 +1,355 @@
+  - task: guest-package-build-osconfig
+    file: guest-test-infra/concourse/tasks/package-integration-test.yaml
+    vars:
+      repo-name: "osconfig"
+- name: guest-diskexpand-integration-test
+  plan:
+  - get: guest-diskexpand
+  - get: guest-test-infra
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: guest-package-build-guest-diskexpand
+    file: guest-test-infra/concourse/tasks/package-integration-test.yaml
+    vars:
+      repo-name: "guest-diskexpand"
+- name: guest-configs-integration-test
+  plan:
+  - get: guest-configs
+  - get: guest-test-infra
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: guest-package-build-guest-configs
+    file: guest-test-infra/concourse/tasks/package-integration-test.yaml
+    vars:
+      repo-name: "guest-diskexpand"
+
+- name: build-guest-agent
+  plan:
+  - get: guest-agent
+    passed: [guest-agent-integration-test]
+    trigger: false
+    params:
+      skip_download: "true"
+  - load_var: commit-sha
+    file: guest-agent/.git/ref
+  - get: guest-test-infra
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: get-github-token
+    file: guest-test-infra/concourse/tasks/get-github-token.yaml
+  - task: generate-package-version
+    file: guest-test-infra/concourse/tasks/generate-package-version.yaml
+    vars:
+      repo-name: "guest-agent"
+  - load_var: package-version
+    file: package-version/version
+  - in_parallel:
+      fail_fast: true
+      steps:
+      - task: guest-package-build-guest-agent-deb9
+        file: guest-test-infra/concourse/tasks/guest-package-build.yaml
+        vars:
+          wf: "build_deb9.wf.json"
+          repo-name: "guest-agent"
+          gcs_path: "gs://gcp-guest-package-uploads/guest-agent"
+      - task: guest-package-build-guest-agent-el7
+        file: guest-test-infra/concourse/tasks/guest-package-build.yaml
+        vars:
+          wf: "build_el7.wf.json"
+          repo-name: "guest-agent"
+          gcs_path: "gs://gcp-guest-package-uploads/guest-agent"
+      - task: guest-package-build-guest-agent-el8
+        file: guest-test-infra/concourse/tasks/guest-package-build.yaml
+        vars:
+          wf: "build_el8.wf.json"
+          repo-name: "guest-agent"
+          gcs_path: "gs://gcp-guest-package-uploads/guest-agent"
+      - task: guest-package-build-guest-agent-goo
+        file: guest-test-infra/concourse/tasks/guest-package-build.yaml
+        vars:
+          wf: "build_goo.wf.json"
+          repo-name: "guest-agent"
+          gcs_path: "gs://gcp-guest-package-uploads/guest-agent"
+  - task: validate-version
+    file: guest-test-infra/concourse/tasks/validate-version.yaml
+    vars:
+      repo-name: "guest-agent"
+  - task: tag-commit
+    file: guest-test-infra/concourse/tasks/tag-commit.yaml
+    vars:
+      repo-name: "guest-agent"
```

Comment:
it seems like we would need the commit to tag?

Reply:
you resolved this without comment. we do have to pass commit-sha here, and in all other invocations of this task

Reply:
Pass `tag` and `sha` as vars